app/controllers/case_management/anonymized_intake_csv_extracts_controller.rb

```diff
@@ -0,0 +1,18 @@
+module CaseManagement
+  class AnonymizedIntakeCsvExtractsController < ApplicationController
+    layout "admin"
+
+    load_and_authorize_resource
```

Comment:
It's cool that these lines:

```
    if user.is_admin
      can :manage, :all
    end
```

mean that `load_and_authorize_resource` works magically.

Reply:
I totally agree, I was happy about that! Less work for me. And makes me feel confident that we can add pages for admins and not worry about the wrong people see it because we'd need to add it explicitly to their list of permissions